test/tests.sh

```diff
@@ -498,7 +498,9 @@ test_config()
     wd -q --config "$arg_config" add
 
     assertEquals 1 "$(wcl < "$arg_config")"
-    assertEquals "$wd_config_lines" "$(total_wps)"
+    assertEquals "$wd_config_lines" "$(total_wps)" # $WD_CONFIG is unchanged
+    assertEquals "$wd_config_lines" 0
```

Comment:
🏷️ No change to this part of the test, just a note with what's now being tested and `0` to make this part a little more clear!